### ISSUE TITLE
How to debug and see the parsing process trace of chumsky combinators. 

### ISSUE THREAD

**Issue by baoyachi** [zesterer/chumsky#625]
I flipped through the documents and found that the current status is TODO.
* https://github.com/zesterer/chumsky/blob/main/guide/debugging.md

**Comment by zesterer** [comment]
Yes, unfortunately this is not implemented for 1.0 yet. If you have thoughts about features you'd like to see, feel free to throw them in #19!

**Comment by baoyachi** [comment]
@zesterer Thank you for your prompt reply.
